### ISSUE TITLE
Make Table of Todos (Read)

### ISSUE THREAD

**Issue by Neil-G** [Neil-G/todoExample#3]
- [x] make a semantic ui table for the todos
- [ ] initialize the app with a single todo (in initialStoreState.js), and have it render in the table

**Comment by Neil-G** [comment]
The schema for a todo is:

```javascript
{
  title: str, 
  description: str,
  completed: bool
}

```

**Comment by ryanyeh-code** [comment]
do i make a separate component just for todos and a separate component for the table?

**Comment by Neil-G** [comment]
nah, you can just write everything in ```/TodoPage.js``` and extract stuff as need.  For now just make a table with the header fields title, description, and completed.  

This link will give you some instructions on semantic UI tables
https://semantic-ui.com/collections/table.html

Keep in mind also that class in html is className in React- so use className instead of class

Lastly, you can always push up code to your new branch and make a PR and I can then review the code and leave comments directly on pieces of code

**Comment by Neil-G** [comment]
We're going to populate the table in these steps:

- [ ]  manually make a row in the table, on the TodosPage.js itself
- [ ] connect the todos page to the store using mastermind.connectStore, and feed it the todos branch from the store
- [ ] add an initial todo in initialStoreState.js
- [ ] render this initial todo in the table
